src/main/js/components/table/Table.jsx

```diff
@@ -73,12 +80,57 @@ class Table extends React.Component {
     Dispatcher.off(ActionTypes.SHOW_ROW_CONTEXT_MENU, tableContextMenu.showRowContextMenu, this);
     Dispatcher.off(ActionTypes.CLOSE_ROW_CONTEXT_MENU, tableContextMenu.closeRowContextMenu, this);
     Dispatcher.off(ActionTypes.DUPLICATE_ROW, tableRowsWorker.duplicateRow, this);
+    Dispatcher.off(ActionTypes.COPY_CELL_CONTENT, this.setCopyOrigin);
+    Dispatcher.off(ActionTypes.PASTE_CELL_CONTENT, this.pasteCellTo);
 
     window.removeEventListener("resize", this.windowResize);
     this.props.table.rows.off("add", tableRowsWorker.rowAdded.bind(this));
     window.GLOBAL_TABLEAUX.tableRowsDom = null;
   }
 
+  setCopyOrigin = cell => {
+    this.pasteOriginCell = cell;
+  };
+
+  pasteCellTo = ({cell}) => {
+    const src = this.pasteOriginCell.cell;
+    if (cell.kind === "link" && !(cell.column.id === src.column.id)) { // only copy same cell type or links from same column
```

Comment:
Is `pasteOriginCell` going to be unset on table switch? Checking just the column ID could be a problem.